inc/woocommerce.php

```diff
@@ -16,7 +16,18 @@
  * @return void
  */
 function _s_woocommerce_setup() {
-	add_theme_support( 'woocommerce' );
+	add_theme_support( 'woocommerce', array(
+		'thumbnail_image_width' => 150,
+		'single_image_width'    => 300,
+		'product_grid'          => array(
+			'default_rows'    => 3,
+			'min_rows'        => 2,
+			'max_rows'        => 8,
+			'default_columns' => 4,
+			'min_columns'     => 2,
```

Comment:
```suggestion
			'min_columns'     => 1,
```
There should be no reason not to allow a single column. This is the WooCommerce default.